test-framework/test-suites/integration/tests/list/test_list_attr.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import json
+
+@pytest.mark.usefixtures('revert_database')
+class TestListAttr:
+
+
+	def test_list_attr_with_shadow(self, host):
+		result = host.run('stack list attr output-format=json')
+		assert result.rc == 0
+
+		result = host.run('stack list attr shadow=True output-format=json')
+		assert result.rc == 0
```

Comment:
This is only testing that the commands both returned `0`.  That's not nothing (at least we know they don't stacktrace) but I think we can put some further checks here?

For example, the output between `shadow=True` and `shadow=False` should be different.  Another test would be to ensure that `stack list attr` by itself has the exact same output as `shadow=False` (the default).